src/App.js

```diff
@@ -1,20 +1,38 @@
- import React, {Component} from 'react';
+import React, {Component} from 'react';
 import Header from './Header';
 import Footer from './Footer';
 import MovieContainer from './MovieContainer';
-import movieData from './movieData';
-import './App.css';
 import MovieDetails from './MovieDetails';
+import './App.css';
 
 class App extends Component {
   constructor(){
     super();
     this.state = { 
-      movies: movieData.movies,
+      movies: [],
+      error:'',
       isClicked: false,
       selectedMovie: {}
     }
   }
+
+  componentDidMount = () => {
```

Comment:
Really nice work setting up your fetch request in `componentDidMount`.  I love that your **throwing a new Error** if the response isn't okay and that you're setting the error message to state.  The only small refactor I might suggest if you have time is to move part of your fetch request into an `apiCalls.js`.  Great work through!